src/Illuminate/Mail/MailManager.php

```diff
@@ -213,40 +191,36 @@ protected function createSmtpTransport(array $config)
     /**
      * Configure the additional SMTP driver options.
      *
-     * @param  \Swift_SmtpTransport  $transport
+     * @param  \Symfony\Component\Mailer\Transport\Smtp\EsmtpTransport  $transport
      * @param  array  $config
-     * @return \Swift_SmtpTransport
+     * @return \Symfony\Component\Mailer\Transport\Smtp\EsmtpTransport
      */
-    protected function configureSmtpTransport($transport, array $config)
+    protected function configureSmtpTransport(EsmtpTransport $transport, array $config)
     {
-        if (isset($config['stream'])) {
-            $transport->setStreamOptions($config['stream']);
```

Comment:
It seems like the biggest use case for the stream options are disabling the verification of SSL-Certificates. This is because in enterprise networks there are self-signed certificates which needs to be trusted.
https://stackoverflow.com/questions/60329775/how-to-disable-verify-peer-with-symfony-mailer-component

The same behaviour in the Symfony Mailer Component can be achieved with appending `?verify_peer=false` to the DSN String.
This was released with Symfony Mailer v5.1 https://github.com/symfony/symfony/pull/35262

This should probably be mentioned in the upgrade-guide if some applications are relying on disabling the verification. (Which should be avoided if possible because it introduces Man in the Middle Attack Scenarios)

Reply:
Thanks @Jubeki, I've pushed a commit that allows apps to set options on the ESMTP transport through the config so they should now be able to set these new verify_peer options.